scripts/falco-driver-loader

```diff
@@ -506,13 +482,14 @@ print_usage() {
 	echo "Options:"
 	echo "  --help         show brief help"
 	echo "  --clean        try to remove an already present driver installation"
-	echo "  --compile      try to compile the driver locally"
-	echo "  --download     try to download a prebuilt driver"
+	echo "  --compile      try to compile the driver locally (default on)"
```

Comment:
```suggestion
	echo "  --compile      try to compile the driver locally (default true)"
```

Reply:
Thanks for the suggestions. I'm gonna use "yes" which is the default value when both flags are not passed. It's also the value that gets assigned to `ENABLE_COMPILE` when `--compile` is rather passed in by the user (same for `--download`).